spec/classes/foreman_proxy_content__broker_spec.rb

```diff
@@ -7,7 +7,9 @@
         facts
       end
 
-      it { is_expected.to compile.with_all_deps }
```

Comment:
Given this example, normally I'd be cleaner to do modify line 5:
```ruby
context "on #{os}", if: facts[:operatingsystemmajrelease] == '7'
```

That skips the entire context and produces less output.